src/components/ShareTrajectoryModal/style.css

```diff
@@ -0,0 +1,106 @@
+.upload-modal :global(.ant-modal-footer) {
+    padding: 12px 16px 20px;
+    background-color: var(--modal-content-bg);
+    max-height: 56px;
+}
+
+.upload-modal :global(.ant-modal-body) {
+    padding: 24px 24px 0px 24px;
+    font-size: 16px;
+    color: var(--dark);
+}
+
+.upload-modal :global(.ant-modal-header) {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.upload-modal :global(.ant-modal-title) {
+    font-size: 14px;
+}
+
+.upload-modal :global(.ant-modal-close-x) {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.upload-modal :global(.ant-checkbox-inner) {
+    background-color: transparent;
+    border: 1px solid var(--dark-three) !important;
+}
+
+input {
+    border-radius: 10px;
```

Comment:
this is going to style every input with a border radius. If we want this, it should be in the general style sheet. Otherwise here it needs to be wrapped in a class